docker/container.go

```diff
@@ -379,26 +381,12 @@ func holdHijackedConnection(tty bool, inputStream io.ReadCloser, outputStream, e
 	return nil
 }
 
-// Up creates and start the container based on the image name and send an event
-// to notify the container has been created. If the container exists but is stopped
-// it tries to start it.
-func (c *Container) Up(ctx context.Context, imageName string) error {
```

Comment:
This did not make sense at all (a container do not `Up` itself) and the change was easy enough to make in the PR (it even make the code path easier to follow :angel: ).